### PR TITLE
chore: revamp onboarding flow

### PR DESCRIPTION
**Pull request by tanmoysrt**
https://github.com/user-attachments/assets/59c5b0ba-ec72-4570-9b04-0ae5c99d7058

### CODE REVIEW

File: dashboard/src/components/UpdateBillingDetailsForm.vue [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+			@click="$resources.updateBillingInformation.submit"
+		>
+			Update Billing Details
+		</Button>
+	</div>
+</template>
+
+<script>
+import AddressForm from '@/components/AddressForm.vue';
+import { notify } from '@/utils/toast';
```
Comment (BreadGenie) [inline dashboard/src/components/UpdateBillingDetailsForm.vue]: We are using toast from vue-sonner instead of the old toast. Can you change this?

Comment (BreadGenie) [reply dashboard/src/components/UpdateBillingDetailsForm.vue]: On 2nd look this file seems to be in src folder can you move this to src2?
Having 2 src folders must've confused you. I'll remove the older files soon.

Comment (tanmoysrt) [reply dashboard/src/components/UpdateBillingDetailsForm.vue]: okay

Comment (tanmoysrt) [reply dashboard/src/components/UpdateBillingDetailsForm.vue]: Moved those pages to src2

**Pull request merged**